src/ngx_http_lua_regex.c

```diff
@@ -1922,12 +1924,78 @@ ngx_http_lua_ngx_re_sub_helper(lua_State *L, unsigned global)
 }
 
 
+ngx_int_t
+ngx_http_lua_set_jit_stack_size(int size)
```

Comment:
This C function should be guarded by `LUA_HAVE_PCRE_JIT`. Better test with PCRE builds without JIT enabled to be sure here.